runners/jstorm/src/main/java/org/apache/beam/runners/jstorm/TestJStormRunner.java

```diff
@@ -116,6 +117,30 @@ public JStormRunnerResult run(Pipeline pipeline) {
     }
   }
 
+  private void handleTaskException(Throwable taskExceptionRec) {
+    Throwable cause;
+    if (taskExceptionRec.getCause() != null) {
+      cause = taskExceptionRec.getCause();
+    } else {
+      cause = taskExceptionRec;
+    }
+
+    UserCodeException innermostUserCodeException = null;
+    Throwable current = cause;
+    for (; current.getCause() != null; current = current.getCause()) {
+      if (current instanceof UserCodeException) {
+        innermostUserCodeException = ((UserCodeException) current);
+      }
+    }
+    if (innermostUserCodeException != null) {
+      cause = innermostUserCodeException.getCause();
+    }
+    if (current instanceof AssertionError) {
```

Comment:
Should it be "cause" instead of "current"?

Reply:
Done